bedrock/firefox/templates/firefox/features.html

```diff
@@ -0,0 +1,72 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox')}} — {% endblock %}
+{% block page_title %}{{ _('Features') }}{% endblock %}
+{% block page_desc %}{{ _('Get the most from Firefox and your time online with Hello, Sync and Private Browsing.') }}{% endblock %}
+{% block body_id %}firefox-features{% endblock %}
+{% block body_class %}mozid{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_features' %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+{% block site_header_unwrapped %}
+  {{ fxfamilynav('features') }}
+{% endblock %}
+
+{% block content %}
+
+<main id="main-content">
+  <div id="main-feature">
+    <h1>{{ _('Firefox Features') }}</h1>
+
+    <p>
+      {{ _('Learn how you can get the most from Firefox and your time online.') }}
+    </p>
+  </div>
+
+  <ul id="fxfeature-list">
+    <li class="private-browsing">
+      <h2>{{ _('Private Browsing') }}</h2>
+
+      <p>
+        {{ _('No other browser gives you so much control over your digital footprint.') }}
+      </p>
+
+      <a href="{{ url('firefox.private-browsing') }}">{{ _('Learn more about Private Browsing') }}</a>
```

Comment:
Not sure how much design guidance we got for this page, but I might suggest embiggening these links or even wrapping the link around the entire block (which would mean some other wrapper with a class for the cta text).